keyboards/bastardkb/charybdis/3x5/info.json

```
@@ -6,7 +6,7 @@
         "vid": "0xA8F8"
     },
     "layouts": {
```

Pretty sure the aliases should be here:
```suggestion
    "layout_aliases": {
    	"LAYOUT_charybdis_3x5": "LAYOUT"
    }
    "layouts": {
```

---

Done.